src/Routing/Route/Route.php

```diff
@@ -423,6 +423,7 @@ public function match(array $url, array $context = [])
             $hostOptions['_ext'] = $url['_ext'];
             unset($url['_ext']);
         }
+        unset($defaults['_ext']);
```

Comment:
I guess I'm confused how routes connected by resources differ from routes connected with `connect()` once an extension has been connected. Ideally there would be no difference.